app/helpers/application_helper.rb

```diff
@@ -208,8 +208,7 @@ def admin_block(ontology: @ontology, user: session[:user], class_css: "admin-bor
   end
 
   def subscribed_to_ontology?(ontology_acronym, user)
-    user.bring(:subscription) if user.subscription.nil?
-    # user.subscription is an array of subscriptions like {ontology: ontology_id, notification_type: "NOTES"}
+    user = LinkedData::Client::Models::User.find(user.username, {include: 'subscription'})
```

Comment:
why this ?

Reply:
user.bring(:subscription) get always nil

Reply:
the function `bring` does not exist in the User model,  

here you should change it to
```
 user = LinkedData::Client::Models::User.find(user.username, {include: 'subscription'}) if user.subscription.nil?
```